release/release_workflow.wdl

```diff
@@ -106,12 +107,14 @@ task versionPrep {
     }
 
     command <<<
-      set -e
+        set -euxo pipefail
 
-      which jq || brew install jq
-      curl --fail -v -s https://api.github.com/repos/~{organization}/cromwell/releases/latest | jq --raw-output '.tag_name' > version
+        which jq || brew install jq
+        # Latest is the latest release by time, not version number.
+        # Instead, grab the first page of releases, then find the max release version from the returned results
+        curl --fail -v -s https://api.github.com/repos/~{organization}/cromwell/releases | jq --raw-output 'max_by(.tag_name)|.tag_name' > version
```

Comment:
Since we're going to be releasing more frequently here's some Cromwell 100 ready jq: 

```
# https://rosettacode.org/wiki/Determine_if_a_string_is_numeric
def is_numeric: true and try tonumber catch false; [ .[] | select(.tag_name|is_numeric) ] | max_by(.tag_name|tonumber) | .tag_name
```